server/src/test/java/io/littlehorse/common/model/metadatacommand/PrincipalAdministrationTest.java

```diff
@@ -209,21 +219,6 @@ public void supportPrincipalDeletion() {
                 .isNull();
     }
 
-    @Test
-    public void shouldRestrictPrincipalCreationToCurrentTenant() {
```

Comment:
This test prevents the creation of a principal that has `perTenantAcls` differents from the requester principal's `perTenantAcls` Could you please explain why this test should be removed?

Reply:
If a requester has perTenantAcls for a tenant, and it is making requests with that tenants client_id, then the request should be rejected entirely. It should not be able to access cluster-scoped resources (Principals or Tenants) at all.

Maybe I am misunderstanding the purpose of this test, but it seemed to not make sense in this new permissions context.

Reply:
Then this test should change, we need to validate that behavior that you just described.

Reply:
Precisely, I believe the 2 tests I proposed here https://github.com/littlehorse-enterprises/littlehorse/pull/1032#issuecomment-2377371915 should target that scenario.

Reply:
I see, yes. This test case should be replaced with those two tests.